src/sidebar/components/moderation/ModerationControl.tsx

```diff
@@ -0,0 +1,38 @@
+import { ModerationStatusSelect } from '@hypothesis/annotation-ui';
+
+import type { Annotation } from '../../../types/api';
+import ModerationStatusBadge from './ModerationStatusBadge';
+
+export type ModerationControlProps = {
+  annotation: Annotation;
+  groupIsPreModerated: boolean;
+  /** Classes to apply to the ModerationStatusBadge in case it's rendered */
+  badgeClasses?: string | string[];
+};
+
+export default function ModerationControl({
+  annotation,
+  groupIsPreModerated,
+  badgeClasses,
+}: ModerationControlProps) {
+  const canModerate = annotation.actions?.includes('moderate');
+  const moderationStatus = annotation?.moderation_status ?? 'APPROVED';
+
+  // We don't want to show any moderation control for approved annotations in
+  // non-pre-moderated groups
```

Comment:
Probably worth re-adding the note here from earlier PRs about the rationale (to avoid cluttering the UI when most annotations are approved).